src/transformations.jl

```diff
@@ -8,16 +8,26 @@ abstract type AbstractDataTransform end
 
 Apply transformation `t` to vector or matrix `x` in place.
 """
-transform!(t::AbstractDataTransform, x::AbstractArray{<:Real,1}) = transform!(x, t, x)
-transform!(t::AbstractDataTransform, x::AbstractArray{<:Real,2}) = transform!(x, t, x)
+function transform!(t::AbstractDataTransform, x::AbstractVecOrMat{<:Real}; dims::Union{Integer,Nothing}=nothing)
+    if dims == 1
```

Comment:
When `dims` defaults to `nothing`, you should call `depwarn` with some deprecation warning and set `dims` to 2

Reply:
OK